vcd/datasource_vcd_org_test.go

```diff
@@ -0,0 +1,89 @@
+// +build org ALL functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+// Cloning an organization using an existing organization as data source
+func TestAccVcdDatasourceOrg(t *testing.T) {
+
+	if !usingSysAdmin() {
```

Comment:
I am not sure, but for me seems that should be valid case where admin org user still should be able access orgs within his rights.

Reply:
The test creates another Org. For this, it requires Syestem Admin rights